Jiti Breaks Mesh Internal Codegen

### Issue workflow progress

<!-- PLEASE DO NOT REMOVE THIS SECTION -->

_Progress of the issue based on the
[Contributor Workflow](https://github.com/the-guild-org/Stack/blob/master/CONTRIBUTING.md#a-typical-contributor-workflow)_

- [ ] 1. The issue provides a reproduction available on
     [Github](https://github.com/ardatan/graphql-mesh/tree/master/examples/hello-world),
     [Stackblitz](https://stackblitz.com/github/ardatan/graphql-mesh/tree/master/examples/hello-world)
     or
     [CodeSandbox](https://codesandbox.io/s/github/ardatan/graphql-mesh/tree/master/examples/hello-world)

> Make sure to fork this template and run `yarn generate` in the terminal.
>
> Please make sure Mesh package versions under `package.json` matches yours.

- [ ] 2. A failing test has been provided
- [ ] 3. A local solution has been provided
- [ ] 4. A pull request is pending review

---

**Describe the bug**

I have an issue with the update to @graphql-mesh/cli@0.92.0 and higher. I am unable to build my application which has tsconfig paths. When I run `mesh build` I get a jiti error that looks like this:

```
[2024-07-29T13:16:56.224Z] 🕸️  Mesh 💥 Error: Cannot find module '@config'
Require stack:
- /project/path/src/graphql/envelop-plugins.ts
    at Module._resolveFilename (node:internal/modules/cjs/loader:1219:15)
    at Function.resolve (node:internal/modules/helpers:133:19)
    at _resolve (/project/path/node_modules/jiti/dist/jiti.js:1:241814)
    at jiti (/project/path/node_modules/jiti/dist/jiti.js:1:244531)
    at /project/path/src/graphql/envelop-plugins.ts:1:252
    at evalModule (/project/path/node_modules/jiti/dist/jiti.js:1:247313)
    at jiti (/project/path/node_modules/jiti/dist/jiti.js:1:245241)
    at /project/path/node_modules/jiti/dist/jiti.js:1:248272
    at Generator.next (<anonymous>)
    at /project/path/node_modules/jiti/dist/jiti.js:1:238153
```
this output is cleaned to remove sensitive info

As you can see there's an issue when modules are named using typescript aliases. I've narrowed it down to the codegen implementation in mesh. It is not able to traverse typescript aliases anymore following this update to add jiti. This doesn't happen on 0.91.2 for me so that's the version I'm using right now, but I'll be blocked from updating the cli package otherwise.

**To Reproduce** Steps to reproduce the behavior:

I'm able to provide a repro repo if necessary. But I believe these steps should suffice if you can make them yourself.

- Create a valid .meshrc.yaml file 
- add `codegen` block with a `contextType`
- import a type value from a local that uses a TypeScript path alias.

**Expected behavior**

Successful build. 

Instead, we get an error saying: `Mesh 💥 Error: Cannot find module '<typescript path alias>'`

**Environment:**

- OS:
- `@graphql-mesh/cli@0.92.x`:
- NodeJS: LTS (20.16)

**Additional context**

I was not successful in replicating this with did need to add envelop plugins

--- a/.changeset/shaggy-cycles-eat.md
+++ b/.changeset/shaggy-cycles-eat.md
@@ -1,0 +1,5 @@
+---
+'@graphql-mesh/cli': patch
+---
+
+No default importFn for findAndParseConfig, making the processor use utils importFn

--- a/packages/legacy/cli/src/config.ts
+++ b/packages/legacy/cli/src/config.ts
@@ -55,7 +55,7 @@ export async function findAndParseConfig(options?: ConfigProcessOptions) {
     configName = 'mesh',
     dir: configDir = '',
     initialLoggerPrefix = '🕸️  Mesh',
-    importFn = include,
+    importFn,
     ...restOptions
   } = options || {};
   const dir = path.isAbsolute(configDir) ? configDir : path.join(process.cwd(), configDir);

